components/Navbar.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import Link from "next/link";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faHouse} from "@fortawesome/free-solid-svg-icons";
+import {faRoute} from "@fortawesome/free-solid-svg-icons";
+import {faBars} from "@fortawesome/free-solid-svg-icons";
+
+export function Navigation() {
+  return (
+    <Navbar>
+      <NavList>
+        <ListItems>
+          <Link href="/">
+            <FontAwesomeIcon icon={faHouse} alt="home icon" />
```

Comment:
The Lighthouse test still fails - `<FontAwesomeIcon>` renders an `<svg>` tag and does not support the `alt` attribute. In a pure `<svg>` tag you could include a `<title>` tag, but I am not sure if `<FontAwesomeIcon>` supports passing this down to the rendered output. Please do a little bit of research regarding that problem - if you can't find a solution within 15 minutes we would ignore this for now...

Reply:
Hi Ralf, I'm not so used to lighthouse yet and I'm not sure where exactly the test fails. Unfortunately no one in my breakout-room could help me either.  I tried to look up solutions for the past hour, but can't seem to find a proper solution for this specific problem. I now tried it with the <title> and pushed again.  But since I didn't make any progress from yesterday yet, I would love to work on my next feature for now, instead of investing more time in this accessability topic.